README.md

```diff
@@ -76,250 +80,50 @@ Once you have created your makers, run `:Neomake` as normal. Run
 filetype to use makers will currently cause the `makeprg` to be ignored (this
 should be remedied).
 
-## Plugin documentation
-
-For more detailed documentation please refer to the
-[plugin's help](https://github.com/neomake/neomake/tree/master/doc/neomake.txt)
-(`:h neomake`).
-
-## Included makers
-
-This list may be out of date, look at
-[autoload/neomake/makers](https://github.com/neomake/neomake/tree/master/autoload/neomake/makers)
-for all supported makers.
-
-Applescript:
-
-- osacompile
-
-C:
-
-- clang
-- gcc
-- clang-tidy
-- checkpatch
-
-C++:
-
-- clang++
-- g++
-- clang-tidy
-
-CFEngine 3:
-
-- cf-promises
-
-CUDA:
-
-- nvcc
-
-Coffeescript:
-
-- [coffeelint](http://www.coffeelint.org/)
-
-CSS:
-
-- csslint
-- [stylelint](http://stylelint.io/)
-
-D:
-
-- dmd
-
-Elixir:
-
-- credo (not enabled by default)
-- dogma (not enabled by default)
-- elixirc
-
-Erlang:
-
-- erlc
-
-fish:
-
-- fish
-
-Fortran
-
-- gfortran
-- ifort
-
-Go:
-
-- go
-- golint
-- go vet
-
-Haskell:
-
-- hlint
-- ghc-mod
-- hdevtools
-- cabal
-- liquid (LiquidHaskell)
-
-Stack projects are supported.
-
-Java:
-
-- javac
-
-JavaScript / ECMAScript:
-
-- [eslint](http://eslint.org/)
-- flow
-- [jscs](http://jscs.info/)
-- jshint
-- jsxhint
-- standard
-- xo
-
-JSON:
-
-- [jsonlint](https://github.com/zaach/jsonlint)
-
-JSX:
-
-- [jsxhint](https://github.com/STRML/JSXHint)
-
-Lua:
-
-- luac
-- luacheck
+Refer to the inline documentation `:h neomake.txt` for more.
 
-Markdown:
+### Directory makers
```

Comment:
They are called project makers also, right?

And we include some already, which could be mentioned.